deployment/ansible/development/roles/docker_compose/templates/docker-compose.override.yml.j2

```diff
@@ -67,3 +80,32 @@ services:
       # Use development settings file
       - DJANGO_SETTINGS_MODULE=core.settings.dev_docker
     network_mode: "bridge"
+
+  {% if inasafe_realtime.use_prebuilt_docker %}
+
```

Comment:
@ismailsunni you need to create a similar section for headless, complete with default environment that will make sure it will be running without using `inasafe-headless` repo.

Reply:
Noted.